pages/CheckingTable.js

```diff
@@ -21,8 +17,8 @@ class CheckingPageTable extends AccountTable {
     async changeCheckingTransactionsTablePrevious(){
         await this.checkingTransactionsTablePreviousButton.click()
     }
-    async activateSwitchChecking (){
-        await this.switchChecking.click()
+    async activateSwitchSaving (accountName){
+        await this.clickElement(await this.switch(accountName))
     }
```

Comment:
Why is a method named `activateSwitchSaving()` present in the _CheckingTable_ class?

Reply:
Done